src/Datadog.Trace/AgentHttpHeaderNames.cs

```diff
@@ -38,6 +38,11 @@ internal static class AgentHttpHeaderNames
         /// </summary>
         public const string ContainerId = "Datadog-Container-ID";
 
+        /// <summary>
+        /// Tells the agent whether top-level spans are computed by the tracer
+        /// </summary>
+        public const string ComputedTopLevel = "Datadog-Client-Computed-Top-Level";
```

Comment:
We can't change the header name, but would you be ok with renaming the property for clarity?
```suggestion
        public const string ComputedTopLevelSpan = "Datadog-Client-Computed-Top-Level";
```

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1303/commits/87ab80ddc9107b9b6cff3028a319b5824358d6e7